### DISCUSSION TITLE
garmin venu 3s

### DISCUSSION THREAD

**Discussion by starschild86** [house-of-abbey/GarminHomeAssistant#95] [General]
hi, i'm looking to buy a garmin venu 3s and was wondering if you could tell me if you know how it's supported.
any experience / comment would be greatly appreciated.

**Comment by josephabbey** [comment]
Hi, we have tested it in the simulator, and it works. However, you are better off asking on the forum to see if anyone else is using it. https://community.home-assistant.io/t/home-assistant-app-for-garmin/637348
